openvdb/openvdb/python/README.md

```diff
@@ -0,0 +1,87 @@
+# `openvdb`
+
+This project contains python bindings for OpenVDB.
+
+## Dependencies
+
+| Name    | Version |
+| ---     | ---     |
+| cmake   | >=3.15  |
```

Comment:
The CMakeLists defines 3.18 as the minimum version.

Reply:
Yep, this is now outdated. Removed this section.